app/graphql/types/invoices/applied_taxes/object.rb

```diff
@@ -7,9 +7,9 @@ class Object < Types::BaseObject
         graphql_name 'InvoiceAppliedTax'
         implements Types::Taxes::AppliedTax
 
-        field :invoice, Types::Invoices::Object, null: false
-
+        field :applied_on_whole_invoice?, GraphQL::Types::Boolean, null: false
```

Comment:
This is just definition for TS method on FE (it turns into `applidOnWholeInvoice`) so I would return questionmark at the end of the method since this is mostly rails convention and not TS or JS convention